### PR TITLE
Removed html_enc() from RandomDate.pm

### PR DESCRIPTION
**Pull request by gautamkrishnar**
For fixing bug #3563

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
@GuiltyDolphin i accidentially added metadata of the random number instead of the random dates. I think dax is now deploying it to beta. Please fake necessary action to prevent the deploying of the first IA. :anchor:

**Comment by gautamkrishnar** [comment]
Btw i wil update the test file soon

**Comment by gautamkrishnar** [comment]
OMG! All pased...  :smile:  Yeey.... In my codio two tests were failing:
```
duckpan test RandomDate                                                                                              
t/RandomDate.t .. 1/?                                                                                                                                                                
    #   Failed test 'Deep: full ZCI object'                                                                                                                                          
    #   at /home/codio/perl5/perls/perl-5.18.2/lib/site_perl/5.18.2/DDG/Test/Goodie.pm line 54.                                                                                      
    # Using Regexp on $data->{"structured_answer"}{"data"}{"title"}                                                                                                                  
    #    got : 'Jun 26, 4195 9:41:03 AM'                                                                                                                                             
    # expect : (?^:^(?^:[A-Z][a-z]{2}) (?^:\d{1,2}), (?^:\d{1,4}), (?^:\d{1,2}:\d{2}:\d{2} [AP]M)$)                                                                                  
    # Looks like you failed 1 test of 1.                                                                                                                                             
                                                                                                                                                                                     
#   Failed test 'Query: random datetime'                                                                                                                                             
#   at /home/codio/perl5/perls/perl-5.18.2/lib/site_perl/5.18.2/DDG/Test/Goodie.pm line 58.                                                                                          
                                                                                                                                                                                     
    #   Failed test 'Deep: full ZCI object'                                                                                                                                          
    #   at /home/codio/perl5/perls/perl-5.18.2/lib/site_perl/5.18.2/DDG/Test/Goodie.pm line 54.                                                                                      
    # Using Regexp on $data->{"answer"}                                                                                                                                              
    #    got : '1:15:01 AM'                                                                                                                                                          
    # expect : (?^:^(?^:\d{1,2}:\d{2}:\d{2} PT?G)$)                                                                                                                                  
    # Looks like you failed 1 test of 1.                                                                                                                                             
                                                                                                                                                                                     
#   Failed test 'Query: random time'                                                                                                                                                 
#   at /home/codio/perl5/perls/perl-5.18.2/lib/site_perl/5.18.2/DDG/Test/Goodie.pm line 58.                                                                                          
# Looks like you failed 2 tests of 39.                                                                                                                                               
t/RandomDate.t .. Dubious, test returned 2 (wstat 512, 0x200)                                                                                                                        
Failed 2/39 subtests                                                                                                                                                                 
                                                                                                                                                                                     
Test Summary Report                                                                                                                                                                  
-------------------                                                                                                                                                                  
t/RandomDate.t (Wstat: 512 Tests: 39 Failed: 2)                                                                                                                                      
  Failed tests:  15, 19                                                                                                                                                              
  Non-zero exit status: 2                                                                                                                                                            
Files=1, Tests=39,  0 wallclock secs ( 0.03 usr  0.01 sys +  0.28 cusr  0.03 csys =  0.35 CPU)                                                                                       
Result: FAIL                                                                                                                                                                         
[ERROR]  Tests failed! See output above for details                            
```
But everything is fine here! :open_mouth:

**Comment by GuiltyDolphin** [comment]
@gautamkrishnar Is your DuckPAN up-to-date? `duckpan upgrade`.

File: lib/DDG/Goodie/RandomDate.pm [hunk 1]
```diff
@@ -123,7 +123,7 @@ sub build_subtitle {
         ? " between $min and $max"
         : '';
     $type eq 'format'
-        ? "Random date for: " . html_enc($format)
+        ? "Random date for: " . $format
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/RandomDate.pm]: `"Random date for: $format"`

Comment (gautamkrishnar) [inline lib/DDG/Goodie/RandomDate.pm]: :+1:

**Comment by gautamkrishnar** [comment]
@GuiltyDolphin yes it is uptodate:
```
Emptying DuckPAN cache...                                                                                                                                                            
DuckPAN cache emptied                                                                                                                                                                
[NOTICE] You already have latest version (1009) of App::DuckPAN                                                                                                                      
[NOTICE] You already have latest version (1006) of DDG 
```

**Comment by GuiltyDolphin** [comment]
@gautamkrishnar Ah, I'm afraid you'll need to update your dependencies - it could take a while. Do a quick search on the old Duck and you should be able to find the one-liner.

**Comment by GuiltyDolphin** [comment]
@gautamkrishnar Thanks! :dolphin:

**Pull request merged**